include/dijkstra/v4drivingDist.hpp

```diff
@@ -0,0 +1,489 @@
+/*PGR-GNU*****************************************************************
+
+file: driveDist.hpp
+
+Copyright (c) 2015 pgRouting developers
+Mail: project@pgrouting.org
+
+Copyright (c) 2022 Celia Virginia Vergara Castillo
+Copyright (c) 2015 Celia Virginia Vergara Castillo
+vicky at erosion.dev
+
+Copyright (c) 2023 Aryan Gupta
+guptaaryan1010 AT gmail.com
+
+Copyright (c) 2020 The combinations_sql signature is added by Mahmoud SAKR
+and Esteban ZIMANYI
+mail: m_attia_sakri at yahoo.com, estebanzimanyi at gmail.com
```

Comment:
In case you are creating a new file, you need not include the other contributors - just `project at pgrouting.org` and your email is fine.

Reply:
okay sir